deployment/updates/scripts/xml/notifications/2024_03_03_add_kafka_room_entry_notifications.template.xml

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xml>
+    <config>
+        <serviceUrl>@SERVICE_URL@</serviceUrl>
+        <partnerId>-2</partnerId>
+        <clientTag>Stand alone php 1.0.0</clientTag>
+        <curlTimeout>30</curlTimeout>
+        <userAgent>Stand alone php 1.0.0</userAgent>
+    </config>
+    <session>
+        <partnerId>-2</partnerId>
+        <secret>@ADMIN_CONSOLE_PARTNER_ADMIN_SECRET@</secret>
+        <sessionType>2</sessionType>
+    </session>
+
+    <multirequest>
+        <request service="eventNotificationTemplate" action="add" plugin="eventNotification" partnerId="0">
+            <template objectType="KalturaKafkaNotificationTemplate">
+                <name>Room Entry Updated</name>
+                <systemName>KAFKA_ROOM_ENTRY_UPDATED</systemName>
+                <description>Kafka notification template to be sent when room entry updated</description>
+                <automaticDispatchEnabled>1</automaticDispatchEnabled>
+                <eventType>3</eventType>
+                <!-- EventNotificationEventType::OBJECT_CHANGED -->
+                <eventObjectType>1</eventObjectType>
+                <!-- EventNotificationEventObjectType::ENTRY -->
+                <topicName>@TOPIC_NAME@</topicName>

```

Comment:
I would pur @TOPIC_NAME@ statically to be entry.
I know that previously we did this templating but I am not sure this is the right way.